src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoLink.jsx

```diff
@@ -1,80 +1,49 @@
 import React from 'react';
 import classNames from 'classnames';
 import moment from 'moment';
-import {
-  isClinicVideoAppointment,
-  isVideoHome,
-} from '../../../services/appointment';
 import NewTabAnchor from '../../../components/NewTabAnchor';
-import { VIDEO_TYPES } from '../../../utils/constants';
 
-export default function VideoLink({ appointment, hasFacility, isPast }) {
-  if (isPast) {
-    return <span>Video conference</span>;
-  } else if (appointment.videoData.kind === VIDEO_TYPES.gfe) {
-    return (
-      <span>
-        You can join this video meeting using a device provided by VA.
-      </span>
-    );
-  } else if (isClinicVideoAppointment(appointment)) {
-    return (
-      <span>
-        You must join this video meeting from the VA location{' '}
-        {hasFacility ? 'listed below' : 'where the appointment was scheduled'}.
-      </span>
-    );
-  } else if (appointment.videoData.isAtlas) {
-    return (
-      <span>
-        You must join this video meeting from the ATLAS (non-VA) location listed
-        below.
-      </span>
-    );
-  } else if (isVideoHome(appointment)) {
-    const url = appointment.videoData.url;
-    const diff = moment().diff(moment(appointment.start), 'minutes');
+export default function VideoLink({ appointment }) {
```

Comment:
A bunch of the code in here is now in VideoLocation and this component is solely focused on presenting the video link, when it exists.